### ISSUE TITLE
[$500] Android - Distance - No location access prompt after relaunching app when "Only this time" is chosen

### ISSUE THREAD

**Issue by lanitochka17** [Expensify/App#33416]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:** 1.4.15-4
**Reproducible in staging?:** Y
**Reproducible in production?:** Y
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** Applause - Internal Team
**Slack conversation:**

**Isue found when excuting PR** https://github.com/Expensify/App/pull/33157
## Action Performed:
1. . Tap + > Request money
2. Go to Distance
3. Select "Only this time" for location access
4. Kill app
5. Relaunch the app
6. Tap + > Request money
7. Go to Distance

## Expected Result:
App will ask for location access again after killing the app because "Only this time" for location access was selected

## Actual Result:
App does not ask for location access after killing the app

## Workaround:
Unknown

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [x] Android: Native
- [ ] Android: mWeb Chrome
- [ ] iOS: Native
- [ ] iOS: mWeb Safari
- [ ] MacOS: Chrome / Safari
- [ ] MacOS: Desktop

## Screenshots/Videos

Add any screenshot/video evidence

</details> 

https://github.com/Expensify/App/assets/78819774/4b214f06-777f-42c1-bf05-e9988f1d591b

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

<details><summary>Upwork Automation - Do Not Edit</summary>
    <ul>
        <li>Upwork Job URL: https://www.upwork.com/jobs/~015a8185bc88c68849</li>
        <li>Upwork Job ID: 1737842173239119872</li>
        <li>Last Price Increase: 2023-12-28</li>
    </ul>
</details>

**Comment by ikevin127** [comment]
TLDR:
> Android allows 1 minute from the moment the app goes into the background or is killed until the one-time permission is revoked.

The Android one-time permission [documentation](https://developer.android.com/training/permissions/requesting#one-time) says:

> If the user sends your app to the background, your app can continue to access the data for a short period of time.

From the Android [source code](https://cs.android.com/android/platform/superproject/main/+/main:packages/modules/Permission/PermissionController/src/com/android/permissioncontroller/permission/utils/Utils.java;l=216) we can see this "short period of time" is exactly 1 minute:

```java
/**
 * The default length of the timeout for one-time permissions
 */
public static final long ONE_TIME_PERMISSIONS_TIMEOUT_MILLIS = 1 * 60 * 1000; // <- 1 minute

/**
 * The default length to wait before ending a one-time permission session after all processes
 * are dead.
 */
public static final long ONE_TIME_PERMISSIONS_KILLED_DELAY_MILLIS = 5 * 1000;

```

**Comment by laurenreidexpensify** [comment]
@situchan bump for comment ^^

**Comment by situchan** [comment]
This is not bug but default behavior of android system. Let's close
